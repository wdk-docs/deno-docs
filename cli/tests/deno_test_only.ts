Deno.test({
  name: "abc",
  fn() {},
});

Deno.test({
  only: true,
  name: "def",
  fn() {},
});

Deno.test({
  name: "ghi",
  fn() {},
});
