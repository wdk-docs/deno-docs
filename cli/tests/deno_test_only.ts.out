[WILDCARD]running 1 tests
test def ... ok ([WILDCARD])

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out ([WILDCARD])

FAILED because the "only" option was used

